### DISCUSSION TITLE
希望新增treeselect

### DISCUSSION THREAD

**Discussion by lht931942788** [element-plus/element-plus#1709] [Ideas]
希望新增treeselect

**Comment by Ryoui** [comment]
nice

**Comment by tolking** [comment]
我写了个TreeSelect 组件，感兴趣可以 [查看](https://github.com/tolking/element-pro-components)

**Reply by Ryoui** [reply]
提个pr？我没这技术，只会用

**Comment by yangdaichun** [comment]
这个需求比较强烈

**Comment by msidolphin** [comment]
This feature needs to be considered after the virtual tree is completed
